### ISSUE TITLE
Access for Joe Niquette

### ISSUE THREAD

**Issue by joeniquette** [department-of-veterans-affairs/va.gov-team#16924]
# Instructions
1. Fill out **Description**
2. Add the label used by your team (eg: `BAH-526`)
3. Add a comment @department-of-veterans-affairs/vsp-operations to ping the operations team
4. When the issue is closed you will be notified and can continue with on-boarding setup

# Description
- Please provide your Contracting Officer's Representative (COR) name.
   > Jeffrey Hayter
- Please provide your name.
   > Joseph Niquette
- Please Provide your email.
   > joe.niquette@oddball.io
- Please provide the name of your team, your role on that team, and the name of the company you work for.
   > Identity team within VSP. I am a Sr. Security Engineer at Oddball.

- Grant AWS Console access?
  > YES
  - Need additional access please. I am trying to run metrics from cloudwatch from 2019 for SSO. I currently do not have the required AWS permissions to be able to create cloudwatch metrics. If someone could please help me with obtianing approval and granted access to be able to add metrics specifically for: `dsva-vagov-prod/srv/vets-api/src/log/vets-api-server.log` and `dsva-vagov-prod/var/log/nginx/access.log`
- Being granted access to create metrics for these two log groups would allow me to run analytics from 2019 which we cannot currently perform efficiently or accurately through AWS CLI or cloudwatch Logs Insights. 

Thank you for your assistance!

**Comment by tskinn** [comment]
@joeniquette Did Jeremys PR fulfill this request? Can we close this issue?

**Comment by joeniquette** [comment]
Yes @tskinn I believe we are good to go. I have the permissions I need.

**Comment by jbritt1** [comment]
Think there may still be a piece we have to catch up in the code to include a manual change @pnwstevan made to cover a gap I had left. I was going to check in on this and clean it up tomorrow when I return from PTO.

**Comment by jbritt1** [comment]
Looking at this to get it shored up this morning.

**Comment by jbritt1** [comment]
Latest linked PR corrects a group membership and includes the temp permissions given. Closing.
